controllers/token_controller/token_controller.py

```diff
@@ -0,0 +1,75 @@
+import sys
+import enum
+import struct
+from typing import Dict, List, NamedTuple
+from pathlib import Path
+
+# Webots specific library
+from controller import Emitter, Supervisor
+
+# Root directory of the SR webots simulator (equivalent to the root of the git repo)
+REPO_ROOT = Path(__file__).resolve().parent.parent.parent
+
+sys.path.insert(1, str(REPO_ROOT / 'modules'))
+
+from sr.robot.utils import get_robot_device  # isort:skip
+
+
+BROADCASTS_PER_SECOND = 10
+
+
+class Owner(enum.IntEnum):
+    ZONE_0 = 0
+    ZONE_1 = 1
+
+
+class Token(NamedTuple):
+    owner: Owner
+    id: int  # noqa: A003
+
+    def __repr__(self) -> str:
+        return f'<{self.owner.name}:{self.id}>'
+
+
+TOKENS: List[Token] = [
```

Comment:
Is there any way we can derive this from the arena rather than hardcoding?

Reply:
My intention was to move this into `shared_utils.py` so it can be used by this and `radio.py`